Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Allocator/Jobs/DriverFeeUpdates/DriverFee.hs

```diff
@@ -277,19 +294,21 @@ getFinalOrderAmount feeWithoutDiscount merchantId transporterConfig driver plan
             pure (bestOffer.finalOrderAmount, Just bestOffer.offerId, bestOffer.offerDescription.title)
       return (feeWithoutDiscount, finalOrderAmount, offerId, offerTitle)
 
-splitPlatformFee :: HighPrecMoney -> HighPrecMoney -> Plan -> DriverFee -> Maybe HighPrecMoney -> [DriverFee]
-splitPlatformFee feeWithoutDiscount_ totalFee plan DriverFee {..} maxMandateAmount = do
+splitPlatformFee :: HighPrecMoney -> HighPrecMoney -> Plan -> DriverFee -> Maybe HighPrecMoney -> Maybe HighPrecMoney -> [DriverFee]
+splitPlatformFee feeWithoutDiscount_ totalFee plan DriverFee {..} maxMandateAmount coinClearedAmount = do
```

Comment:
we can change parameter name `maxMandateAmount` to `maxAmountAllowedPerDriverfee`